operators/aqua/6.5.8/bundle.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=aqua
+LABEL operators.operatorframework.io.bundle.channels.v1=6.5.0
+LABEL operators.operatorframework.io.bundle.channel.default.v1=2022.4.0
```

Comment:
@KoppulaRajender you are getting error for `v4.11` because versions `6.5.*` will be included in index and there is no default channel, since you define `LABEL operators.operatorframework.io.bundle.channel.default.v1=2022.4.0`. Can you put it to `LABEL operators.operatorframework.io.bundle.channel.default.v1=6.5.0`